src/blocks/link/Link.tsx

```diff
@@ -27,11 +28,12 @@ const StyledLink = styled(RouterLink).withConfig({
   ${(props) => props.css || ''}
 `;
 
-const Link: FC<LinkProps> = ({ textProps, ...props }) => {
+const Link: FC<LinkProps> = ({ textProps, newTab, ...props }) => {
   const { mode } = useBlocksTheme();
   return (
     <StyledLink
       mode={mode}
+      {...(newTab ? { target: '_blank', rel: 'noopener noreferrer' } : {})}
```

Comment:
Let's not do this here. link component should accept target prop only. This logic should be handled inside the menuitem cimponent.